x-pack/legacy/plugins/canvas/public/components/workpad_header/workpad_header.tsx

```diff
@@ -53,7 +76,7 @@ export class WorkpadHeader extends React.PureComponent {
     </EuiToolTip>
   );
 
-  _keyHandler = action => {
+  _keyHandler = (action: string) => {
```

Comment:
`action` appears to be a predictable type.

```
type Action = 'EDITING' | 'FULLSCREEN' | ...?

_keyHandler = (action: Action) => {
```

Might chat with @cqliu1 about where these are defined, then enumerate them there as a type.  Otherwise, for now, just create a type here that matches what we're expecting.

Reply:
Spoke with @cqliu1 about it.. seems like there's not a good way right now to grab all the actions. I looked briefly at defining them as a big union type but Catherine and I agreed it might be better to have them all listed as an enum that way you can reference them when adding to the keymap (not just referencing a magic string). Problem with doing that enum is that it might be a really not-fun refactor and I'm not sure we wanna block i18n work for it.

I went ahead and created this issue for it though: https://github.com/elastic/kibana/issues/45073